### ISSUE TITLE
SCM changes view: duplicate repository and buttons

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#133984]
Here:

![image](https://user-images.githubusercontent.com/900690/135077765-8fed89d1-bcf1-4891-ab4e-5ed38e3d738f.png)

I always have the repositories view visible but now I see the same information again because of `"scm.alwaysShowRepositories": true` having changed.

**Comment by eamodio** [comment]
This was always the behavior when there are 2 or more repositories -- unless you've hidden one? But either way this is by-design with the changes here: https://github.com/microsoft/vscode/issues/110882#issuecomment-928784945

**Comment by bpasero** [comment]
This was not "always" the behaviour, in stable I do not see this. I had to change my settings now to get the old behaviour back:

```
"scm.alwaysShowRepositories": false
```

**Comment by eamodio** [comment]
For multiple repositories, that setting shouldn't have any effect, only if you just have one. Unless there is some interaction with other settings that I'm not aware of.

**Comment by bpasero** [comment]
Well it does, see stable for compare:

![image](https://user-images.githubusercontent.com/900690/135210047-fac3e7ec-1b03-4ff9-8c34-e9e02b75fede.png)

**Comment by gjsjohnmurray** [comment]
> Well it does, see stable for compare:

@bpasero is this the extra duplication you're referring to?

![image](https://user-images.githubusercontent.com/6726799/135240445-dd286075-4c13-471f-bc81-bc4e45a5c30f.png)

**Comment by bpasero** [comment]
The entire row I would argue because I already see all that information from the selected repository in the view above.

**Comment by gjsjohnmurray** [comment]
But if you have expanded your second repo (vscode-docs) as well and have scrolled it into view then the SOURCE CONTROL REPOSITORIES section may not be visible.

**Comment by bpasero** [comment]
Yeah I never select more than one repo and don't really like that we actually allow to do so. Related https://github.com/microsoft/vscode/issues/120089 and https://github.com/microsoft/vscode/issues/113803
